core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -912,11 +912,16 @@ public Void visitWindow(WindowNode node, Context context)
                 WindowNode.Function function = entry.getValue();
                 String frameInfo = formatFrame(function.getFrame());
 
+                String orderingInfo = function.getOrderingScheme().map(orderingScheme -> " " + orderingScheme.orderBy().stream()
```

Comment:
Could we use the `formatOrderingScheme()` method?

Reply:
It gives slightly different output. I copied snippet from `formatAggregation`.
`formatOrderingScheme` adds `[` and `]` around

Reply:
I will use this and also update `formatAggregation`